docs/content/docs/dev/datastream/operators/asyncio.md

```diff
@@ -152,8 +152,9 @@ All subsequent `complete` calls will be ignored.
 
 The following three parameters control the asynchronous operations:
 
-  - **Timeout**: The timeout defines how long an asynchronous operation take before it is finally considered failed,
-    may include multiple retry requests if retry enabled. This parameter guards against dead/failed requests.
+  - **Timeout**: The timeout defines the maximum duration from the first invocation to the final completion of an asynchronous operation,
+    This duration may include multiple retry attempts (if retries are enabled) and determines when the operation is ultimately considered failed.
```

Comment:
should we change:
is ultimately considered failed. -> is ultimately considered complete.

as the ### Timeout Handling section says we can call ResultFuture.complete() - implying a non failure completion after timeout.